### PR TITLE
Update c2c specs based on c2c-schema file

### LINKED ISSUE #1116: code-to-cloud-spec.md needs to be updated based on c2c-schema file.
**Description:**
There are many new fields in the c2c-schema in the [module-ballerina-c2c](https://github.com/ballerina-platform/module-ballerina-c2c) repo. The code-to-cloud-spec.md  needs to be updated with these new fields and their description. 

**Suggested Labels:**
code-2-cloud
   
**Code sample that shows issue:**

**Related Issues:**

### PR DESCRIPTION
**Pull request by Thevakumar-Luheerathan**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: c2c/code-to-cloud-spec.md [hunk 1]
```diff
@@ -242,10 +242,11 @@ Contains container image related properties.
 
 |Identifier   	|Description	                            |Default Value   	 |
 |---	        |---	                                    |---	             |
-|name   	    |Name of the container image                |$USER-$MODULE_NAME  |
-|repository   	|Container repository to host the container |""   	             |
+|name   	    |Name of the container image                |"hello"             |
+|repository   	|Container repository to host the container |"ballerina"   	     |
```
Comment (xlight05) [inline c2c/code-to-cloud-spec.md:246]: This is incorrect as we are specifying the default value here, not a sample value

File: c2c/code-to-cloud-spec.md [hunk 2]
```diff
@@ -281,27 +293,29 @@ Contains the secrets required for the application.
...
 Contains external files required for the code.
 
-|Identifier |Description                                        |Example Value                      |
-|---	    |---	                                            |---	                              |
-|file    |Path of the external file            |"resource/file.txt"          |
-|mount_path |Path of the file in the container    |"/home/ballerina/foo/file.txt" |
+|Identifier |Description                                        |Example Value     |
+|---	    |---	                                            |---	           |
+|file    |Path of the external file            |"resource/file.txt"                |
+|mount_path |Path of the file in the container    |"/home/ballerina/foo/file.conf" |
```
Comment (xlight05) [inline c2c/code-to-cloud-spec.md:299]: Why did we chagne from .txt to .conf?

File: c2c/code-to-cloud-spec.md [hunk 3]
```diff
@@ -281,27 +293,29 @@ Contains the secrets required for the application.
...
 
 Contains the properties related to the deployment.
 
-|Identifier            |Description                                             |Default Value    |
+|Identifier            |Description                                              |Default Value     |
 |---	                |---	                                                 |---	            |
-|internal_domain_name   |Name of the internal domain (k8s service name)         |${MODULE_NAME}-svc|
-|min_memory            |Minimum memory allocated to the container             |"100Mi"        |
+|internal_domain_name   |Name of the internal domain (k8s service name)          |${MODULE_NAME}-svc|
+|external_accessible   |External accessibility of the deployment                 |                  |
```
Comment (xlight05) [inline c2c/code-to-cloud-spec.md:317]: we need to specify the default value, I think the value is false, please recheck with code

**Review by xlight05 — CHANGES_REQUESTED** [review]

File: c2c/code-to-cloud-spec.md [hunk 4]
```diff
@@ -242,10 +242,11 @@ Contains container image related properties.
 
 |Identifier   	|Description	                            |Default Value   	 |
 |---	        |---	                                    |---	             |
-|name   	    |Name of the container image                |$USER-$MODULE_NAME  |
-|repository   	|Container repository to host the container |""   	             |
+|name   	    |Name of the container image                |"hello"             |
```
Comment (anuruddhal) [inline c2c/code-to-cloud-spec.md:245]: Is default image name is `hello` for all the projects?

Comment (Thevakumar-Luheerathan) [reply c2c/code-to-cloud-spec.md]: It is mentioned in the [c2c-schema.json](https://github.com/ballerina-platform/module-ballerina-c2c/blob/211bb5d73b8a613d616e327c5f51e0869684f7b0/c2c-extension/src/main/resources/c2c-schema.json#L38). However, I checked with a sample. It doesn't use `hello`.

**Review by xlight05 — APPROVED** [review]

**Review by anuruddhal — APPROVED** [review]

**Pull request merged**